aws-iot-softwarepackage/contract-tests-artifacts/inputs_1.json

```diff
@@ -0,0 +1,17 @@
+{
+  "CreateInputs": {
+    "PackageName": "{{uuid}}",
```

Comment:
I didn't find anywhere uuid refering to - perhaps we just want to generate a random string here

Reply:
Or we can give it a hardcoded string